README.md

```diff
@@ -288,8 +290,9 @@ Stream data into and out of an iRODS data object
 
 **Parameters**
 - path : The url encoded logical path to a data object
-- offset : The offset in bytes into the data object
-- limit : The maximum number of bytes to read
+- offset : The offset in bytes into the data object. Defaults to 0
+- limit : The maximum number of bytes to read or write. Required for GET
```

Comment:
is there a default here?   i guess not.

Reply:
no default. the cpp rest impl will return an error from the std library.

the current impl is relying on clients to read the API requirements (i.e. the statement saying `limit` is required for GET)